### ISSUE TITLE
Parking Postion is temporary disabled for Skoda E

### ISSUE THREAD

**Issue by dieterlind** [TA2k/ioBroker.vw-connect#307]
Hello!

Since some weeks I don't receive parking positions anymore for my Skoda Enyaq.
After searching the log today I found this line:
info: vw-connect.0 (956) Parking Postion is temporary disabled for Skoda E

Any reason for that??

Thanks

**Comment by TA2k** [comment]
Yes Skoda blocked iobroker this is the reason to set the update to 10min and disable park position but GitHub version has parking position enabled

**Comment by dieterlind** [comment]
Thanks for the information.
Updated with Github version and parking position is updated again.
Any reason known for this measure from Skoda??
